conf/modules/fdm_jsbsim.xml

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="fdm_jsbsim" dir="fdm">
+  <doc>
+    <description>
+      JSBSim FDM backend for NPS simulator
+
+      Use the paparazzi-jsbsim package if it is installed, otherwise look for JSBsim under /opt/jsbsim
+    </description>
+    <configure name="JSBSIM_ROOT" value="/opt/jsbsim" description="set root directory for JSBSim library"/>
+    <configure name="JSBSIM_PKG" value="yes|no" description="enable or disable using pkg-config to get library flags"/>
```

Comment:
maybe we can actually rename this to something more meaningful like `FIND_JSBSIM_VIA_PKG_CONFIG`, which would default to yes...?